languages/ruby/lib/oso/polar/host.rb

```diff
@@ -190,6 +210,10 @@ def subspecializer?(instance_id, left_tag:, right_tag:)
         left_index && right_index && left_index < right_index
       end
 
+      def subclass?(left_tag:, right_tag:)
+        get_class(left_tag) <= get_class(right_tag)
```

Comment:
Lol this is the most cursed less-than-or-equal-to operator ever built

Reply:
🤣